ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -461,12 +522,11 @@ describe('HelpSidebar', () => {
         completionTime: Date.now(),
       },
     ];
-    const wrapper = await component();
-    await waitForFakeTimersAndUpdate(wrapper);
+    component();
 
-    expect(extractionStatusIcon(wrapper).prop('style').color).toEqual(
-      colors.asyncOperationStatus.starting
-    );
+    expect(
+      await findRuntimeStatusIcon(RuntimeStatus.RUNNING)
```

Comment:
the previous version wanted `starting` - do you know why?

Reply:
I'm not sure, but the the test title mentions looking for the running icon. Possibly a typo. It looks like it hadn't changed meaningfully in 3 years.

Reply:
ok